# Goal
geometry_msgs/PointStamped start_pose
geometry_msgs/PointStamped end_pose
---
# Result
bool success
---
# Feedback
geometry_msgs/PointStamped current_pose
